datasquare.go

```diff
@@ -9,8 +9,9 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-// ErrUnevenChunks is thrown when non-nil chunks are not all of equal size.
-var ErrUnevenChunks = errors.New("non-nil chunks not all of equal size")
+// ErrUnevenChunks is thrown when non-nil shares are not all of equal size.
+// Note: chunks is synonymous with shares.
+var ErrUnevenChunks = errors.New("non-nil shares not all of equal size")
```

Comment:
Introducing a new variable for `ErrUnevenShares` could indeed enhance clarity. However, it appears that this PR deliberately avoids making breaking changes, which is why there are still functions, methods, and variables containing `Chunck` in their names. The coexistence of identifiers with both `chunck`s and `share`s might lead to confusion, though.

Reply:
Yep agreed. I'd like to completely eliminate chunk in favor of share but it would be breaking to do so for a few places so I left the existing "chunk" and tried to include comments saying chunk == share